src/rt/cpp/cown.h

```diff
@@ -186,6 +206,11 @@ namespace verona::cpp
   template<typename T, typename... Args>
   cown_ptr<T> make_cown(Args&&... ts)
   {
+    static_assert(

```

Comment:
@lukecheeseman  I've added an assert here to stop people conflicting `const` means.

Reply:
That sounds good to me, there isn't really a use case for an always const cown anyway as you should probably use a shared immutable.